core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1808,6 +1809,15 @@ public ServerBuilder http1HeaderNaming(Http1HeaderNaming http1HeaderNaming) {
         return this;
     }
 
+    /**
+     * Sets the duration between logging uncaught exceptions.
+     * @param exceptionReportingInterval - the duration between logging uncaught exceptions.
```

Comment:
- Hyphen is not necessary.
- Could add more context to the description? e.g.

```
Sets the interval between the log messages that summarize the
exceptions not logged by any decorators or services, such as
{@link LoggingService}.

@param exceptionReportingInterval the interval between the summary log messages, or {@link Duration#ZERO} to disable this feature
```

Reply:
As non-negative values also disables this feature I've changed the javadoc as follow: 
```
    /**
     * Sets the interval between the log messages that report the
     * exceptions not logged by any decorators or services, such as
     * {@link LoggingService}.
     * @param unLoggedExceptionReportInterval the interval between the summary log messages,
     * or set {@link Duration#ZERO} or negative duration to disable this feature
     */
```